src/lib/flexbox/api/base.ts

```diff
@@ -123,6 +123,12 @@ export abstract class BaseFxDirective implements OnDestroy, OnChanges {
     return value ? value.trim() : ((element.nodeType === 1) ? 'block' : 'inline-block');
```

Comment:
Not something that was added in this PR, but instead of hardcoding the `1`, you can use the built-in browser constant: `element.nodeType === Node.ELEMENT_NODE`.